code/game/turfs/open/space/space.dm

```diff
@@ -9,7 +9,6 @@ GLOBAL_VAR_INIT(starlight_color, pick(COLOR_TEAL, COLOR_GREEN, COLOR_CYAN, COLOR
 	temperature = TCMB
 	thermal_conductivity = OPEN_HEAT_TRANSFER_COEFFICIENT
 	heat_capacity = 700000
-	var/starlight_source_count = 0
```

Comment:
I would use a comment here to indicate removal. `//var/starlight_source_count = 0 //MONKESTATION REMOVAL: Unused variable, removed to save memory`